rpc/examples/echo/frontend/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "scripts": {
     "prepare": "./etc/prepare.sh",
+    "build": "npx webpack",
```

Comment:
Instead of `npx webpack`, it would be a bit safer to `npm i --save-dev webpack` to get a pinned version in the devDependencies, then you could just:
```js
    "build": "webpack",
```

Reply:
This seems to break stuff in CI: https://github.com/viamrobotics/goutils/pull/214/commits/f01c993645616882b474ea5b895d6706f1f8463d + https://github.com/viamrobotics/goutils/pull/214/commits/72354b728fed3344e5a999d366891be0d30ca9cd

I'm just going to keep it as `npx webpack` for now to mirror the setup that exists. Then hopefully move to vite soon after that.

Reply:
ah dang, agreed that's the best course of action then 👍